dokan/dokanc.h

```diff
@@ -52,10 +52,10 @@ extern BOOL g_DebugMode;
 // DokanOptions->UseStdErr is ON?
 extern BOOL g_UseStdErr;
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || (defined(__GNUC__) && !defined(__CYGWIN__))
```

Comment:
I will see whether I find the time to look into this more closely at the end of the week. Maybe we can make it easier somehow.

Reply:
no, I won't find the time. Sorry!